devicetypes/capabilities/switch-level-capability.src/switch-level-capability.groovy

```diff
@@ -71,7 +71,7 @@ def off() {
 	'off'
 }
 
-def setLevel(value) {
+def setLevel(value, rate = null) {
```

Comment:
I'm not familiar with how these files are used so I'm not sure if this is OK or not

Reply:
I *think* they are capability examples. @bflorian could possibly confirm?

Reply:
I understand them to be examples as well.